html/janus.js

```diff
@@ -2339,9 +2339,9 @@ function Janus(gatewayCallbacks) {
 				};
 				continue;
 			}
-			if((typeof track.add === 'undefined' && track.add === null) &&
-					(typeof track.remove === 'undefined' && track.remove === null) &&
-					(typeof track.replace === 'undefined' && track.replace === null)) {
+			if(track.add == null &&
+					track.remove == null &&
+					track.replace == null) {
```

Comment:
indeed the previous check was wrong but we must address both `undefined` and `null` cases, so it should be:

```js
			if((typeof track.add === 'undefined' || track.add === null) &&
					(typeof track.remove === 'undefined' || track.remove === null) &&
					(typeof track.replace === 'undefined' || track.replace === null)) {
```

Reply:
`x == null` is actually equivalent to `x === undefined || x === null`, but yes, it is not obvious to anyone who hasn't memorized the truth table. I'll update this